app/document_types.rb

```diff
@@ -50,7 +50,7 @@ def url
 
     def rendering_apps
       entry = DocumentTypes.rendering_apps_from_content_store.find { |el| el[:name] == name }
-      entry.to_h[:apps].to_a.compact
+      entry.nil? ? [] : entry.to_h[:apps].to_a.compact
```

Comment:
`nils` are already handled by the `.to_h`, so I don't think this is necessary?

```
irb(main):003:0> nil.to_h[:apps].to_a.compact
=> []
```

Reply:
Good spot, thanks! Not sure why I needed to add it in the first place as removing it seems to work fine 👍